pool_configs/examples/litecoinz.json

```diff
@@ -0,0 +1,92 @@
+{
+    "enabled": false,
+    "coin": "litecoinz.json",
+
+    "address": "",
+    "_comment_address": "a transparent address to send coinbase rewards to and to transfer to zAddress.",
+
+    "zAddress": "",
+    "_comment_zAddress": "a private address used to send coins to tAddress.",
+
+    "tAddress": "",
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "invalidAddress":"",
+    "_comment_invalidAddress": "Invalid addresses will be converted to the above",
+
+    "walletInterval": 2.5,
+    "_comment_walletInterval": "",
+
+    "rewardRecipients": {
+        "": 1.0
+    },
+
+    "tlsOptions": {
+        "enabled": false,
+        "serverKey":"",
+        "serverCert":"",
+        "ca":""
+    },
+
+    "paymentProcessing": {
+        "minConf": 100,
```

Comment:
please make this 10. this is confirmations needed by s-nomp for shielding too. this could make payments take a long time.